README.md

```
@@ -1,2 +1,32 @@
-# playground
+# Ballerina Playground
+
 Ballerina playground is a web based tool which allows trying out language features.
+
+### Code structure
```

markdownlint-cli2 (0.21.0)

[warning] Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)